docs/api-graduation-guidelines.md

```diff
@@ -0,0 +1,31 @@
+This document defines the expectations of how API graduation is handled across the github.com/kubevirt organization. Individual projects within the KubeVirt org may decide to append additional criteria and timeline constraints to the items outlined here, but should not deviate from the broad expectations as defined in this document.
+
+# Graduation Phase Expecations
+
+## Alpha
+* Feature incomplete, meaning the API might be half implemented as functionality lands in phases.
+* Basic testing exists proving that the merged functionality works e2e.
+* No backwards compatibility guarantees meaning API can mutate and be deprecated without warning.
+* Available for early adopters to test and provide initial feedback.
```

Comment:
SHould we require to also let the API be Alpha?

Reply:
yes, the api would be alpha. This doc is focused on api versioning and graduation. Maybe you're referring to features within an api? I'm unsure if i fully understand this question